### ISSUE TITLE
Inverse Kinematics

### ISSUE THREAD

**Issue by ywyw2015** [stanfordroboticsclub/StanfordQuadruped#24]
What is the coordinate system of pupper？What is the sequence of motor control？

**Comment by Nate711** [comment]
The coordinate system is fixed to the frame of the robot and centered at the geometric center between the hip axes. Not sure what you mean by sequence of motor control.

**Comment by artur-iluk** [comment]
More specific question:
how are oriented axes of pupper? Where points X+axis and so on.

**Comment by FaboNo** [comment]
Yes I guess X axis is forward along the main body frame, Y axis perpendicular. However the question is more about the Z axis is oriented up or down?
Thank you for this  astonished work actually
